src/pages/about.jsx

```diff
@@ -64,171 +66,173 @@ const About = () => {
-                        <div class="card">
-                          
-                            <div class="container">
-                                <h2>Akeem Badmus</h2>
-                                <p class="title">Community Lead</p>
-                            </div>
-                        </div>
-                    </div>
-
-                    <div class="column">
-                        <div class="card">
-                            <div class="container">
-                                <h2>Firstname Lastname</h2>
-                                <p class="title">Designation</p>
-                            </div>
-                        </div>
+                <div className="row__options">
+                    <h2 className="row__stack">Product Managers</h2>
+                    <div style={{ display: "flex" }}>
+                        <IoIosArrowBack className="row__icon" />
+                        <IoIosArrowForward className="row__icon" />
                     </div>
                 </div>
-
-                <h2 className="stack">Product Designers</h2>
-                <div class="row">
-                    <div class="column">
-                        <div class="card">
-                            <div class="container">
-                                <h2>Aslam Dhikrullahi</h2>
-                                <p class="title">Design Lead</p>
-                            </div>
-                        </div>
-                    </div>
-
-                    <div class="column">
-                        <div class="card">
-                            <div class="container">
-                                <h2>Opata Ebubechukwu</h2>
-                                <p class="title">Design Lead</p>
-                            </div>
-                        </div>
-                    </div>
-
-                    <div class="column">
-                        <div class="card">
-                            <div class="container">
-                                <h2>Ojieh Miracle</h2>
-                                <p class="title">Product Designer</p>
-                            </div>
-                        </div>
-                    </div>
-
-                    <div class="column">
-                        <div class="card">
-                            <div class="container">
-                                <h2>Amaefule Ogechi</h2>
-                                <p class="title">Product Designer</p>
-                            </div>
-                        </div>
-                    </div>
-
-                    <div class="column">
-                        <div class="card">
-                            <div class="container">
-                                <h2>Firstname Lastname</h2>
-                                <p class="title">Designation</p>
-                            </div>
-                        </div>
-                    </div>
+                <div className="row">
```

Comment:
Check the design. The cards are supposed to be displayed in a carousel. Research online for libraries that can help with that.